extension/src/main/java/org/creek/system/test/extension/api/model/Input.java

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
-package org.creek.system.test.extension.api;
+package org.creek.system.test.extension.api.model;
```

Comment:
wrong package. 

```suggestion
package org.creek.api.system.test.extension.model;
```

Same for others